exchanges/order/order_types.go

```diff
@@ -133,6 +133,10 @@ type SubmitResponse struct {
 	BorrowSize  float64
 	LoanApplyID string
 	MarginType  margin.Type
+
+	// MultiOrderError is used to indicate an error that occurred when submitting multiple orders, used to determine
+	// rollback point(s) in orders submissions.
+	MultiOrderError error
```

Comment:
Rn -> `SubmissionErorr`/`Error` An individual error doesn't know about other orders sent in a batch. If you are using this for `rollback point(s) in orders submissions.` you should be storing more information about the others inside this struct if its important/making a holding struct instead. But from the sounds of things with the API, this just stores the error for the order and should be renamed

Reply:
I think in this case, vague is better than an ultra specific field that is only for one exchange in one request type